css/soe_helper.css

```
@@ -1595,11 +1595,14 @@ body:not(.page-admin) .sidebar .views-exposed-form .views-exposed-widget .form-s
   .node-type-stanford-news-item .group-s-credits-style.field-group-div.credits {
     display: block; } }
 
+div#block-ds-extras-banner {
```

Don't combine element and ID selectors.
```suggestion
#block-ds-extras-banner {
```